### DISCUSSION TITLE
Health Checks for anycable failing on ALB target groups.

### DISCUSSION THREAD

**Discussion by smasry** [anycable/anycable#140] [Q&A]
I'm attempting to deploy the anycable grpc server in an ALB target group in AWS though the health checks are continuously failing.      Has anyone attempted to deploy this out in a target group that has the health check working correctly that can help point me in the right direction.

**Comment by palkan** [comment]
Could you please provide more details on your ALB configuration? More precisely, `HealthCheckPath` and gRPC code (I'm looking at this tutorial: https://exampleloadbalancer.com/albgrpc_demo.html; don't have an ALB to play with available right now)

**Reply by smasry** [reply]
![image](https://user-images.githubusercontent.com/13121/151429269-70fa28a7-eea5-428f-9c44-d6a87548d97d.png)
We've tried just using '/' as well with a response code of 12

**Reply by palkan** [reply]
Interesting. Looks like ALB doesn't allow setting a service name to check.
I pushed a commit to master to allow an empty service name. Could you please check it?
It that helps, I'll publish a new release.

**Reply by smasry** [reply]
Deployed out the master branch but still not working.  Also set the path to just `/`
What should the path be?
![image](https://user-images.githubusercontent.com/13121/151456306-a257c707-a4ce-4585-8b50-bad68022cb48.png)

**Reply by smasry** [reply]
@palkan this turned out to just be a configuration issue.  It was defaulting to bind to the loopback address.  We updated the host and it's healthy now.  Appreciate you looking into this.
